__tests__/ResourceStateBuilder.test.js

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import ResourceStateBuilder from '../src/ResourceStateBuilder'
```

Comment:
You should be able to use an absolute url here: `import ResourceStateBuilder from 'ResourceStateBuilder'`

Reply:
Fixed.